pkg/app/server/service/webservice/service.proto

```diff
@@ -186,11 +186,26 @@ service WebService {
         option (model.rbac).resource = PROJECT;
         option (model.rbac).action = GET;
     }
-    rpc AddProjectRBACRole(AddProjectRBACRoleRequest) returns (AddProjectRBACRoleResponse) {}
-    rpc UpdateProjectRBACRole(UpdateProjectRBACRoleRequest) returns (UpdateProjectRBACRoleResponse) {}
-    rpc DeleteProjectRBACRole(DeleteProjectRBACRoleRequest) returns (DeleteProjectRBACRoleResponse) {}
-    rpc AddProjectUserGroup(AddProjectUserGroupRequest) returns (AddProjectUserGroupResponse) {}
-    rpc DeleteProjectUserGroup(DeleteProjectUserGroupRequest) returns (DeleteProjectUserGroupResponse) {}
+    rpc AddProjectRBACRole(AddProjectRBACRoleRequest) returns (AddProjectRBACRoleResponse) {
+        option (model.rbac).resource = PROJECT;
+        option (model.rbac).action = CREATE;
+    }
+    rpc UpdateProjectRBACRole(UpdateProjectRBACRoleRequest) returns (UpdateProjectRBACRoleResponse) {
+        option (model.rbac).resource = PROJECT;
+        option (model.rbac).action = UPDATE;
+    }
+    rpc DeleteProjectRBACRole(DeleteProjectRBACRoleRequest) returns (DeleteProjectRBACRoleResponse) {
+        option (model.rbac).resource = PROJECT;
+        option (model.rbac).action = DELETE;
+    }
+    rpc AddProjectUserGroup(AddProjectUserGroupRequest) returns (AddProjectUserGroupResponse) {
+        option (model.rbac).resource = PROJECT;
+        option (model.rbac).action = CREATE;
+    }
+    rpc DeleteProjectUserGroup(DeleteProjectUserGroupRequest) returns (DeleteProjectUserGroupResponse) {
+        option (model.rbac).resource = PROJECT;
+        option (model.rbac).action = DELETE;
```

Comment:
And this.

Reply:
Right, Thank you. https://github.com/pipe-cd/pipecd/pull/3894/commits/220a06bdb28df3a7e6474cf5a3dd16c19614ccce